examples/website/i3s/app-debug.js

```diff
@@ -274,7 +275,17 @@ export default class App extends PureComponent {
     this.setState({tilesetUrl, name, token});
     this.handleClearWarnings();
     const metadata = await fetch(metadataUrl).then((resp) => resp.json());
-    this.setState({metadata, tileInfo: null, normalsDebugData: []});
+    const layerUrls = await this.getLayerUrls(tilesetUrl);
+    this.setState({metadata, tileInfo: null, normalsDebugData: [], layerUrls});
+  }
+
+  async getLayerUrls(tilesetUrl) {
```

Comment:
- JSDoc, typescript

Reply:
It is a react component file. I am not sure we should change it to typescript.
JSDoc added here https://github.com/visgl/loaders.gl/pull/1839